src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceSourceConfig.java

```diff
@@ -200,7 +200,43 @@ public void validate(FailureCollector collector) {
       }
     }
     validateSchema(collector);
-    validatePKChunk(collector);
+    validatePKChunk(collector, oAuthInfo);
+  }
+  public void validateSOQLQueryParameters(FailureCollector collector) {
```

Comment:
Leave a space before the method. Is this new functionality or refactored to move here?

Reply:
Added. These are the validations that need to be done without setting up connection with Salesforce.